packages/components/select/src/select.vue

```diff
@@ -35,81 +35,86 @@
             :class="nsSelect.e('tags')"
             :style="selectTagsStyle"
           >
-            <span
+            <transition
               v-if="collapseTags && selected.length"
-              :class="[
-                nsSelect.b('tags-wrapper'),
-                { 'has-prefix': prefixWidth && selected.length },
-              ]"
+              @after-leave="resetInputHeight"
             >
-              <el-tag
-                :closable="!selectDisabled && !selected[0].isDisabled"
-                :size="collapseTagSize"
-                :hit="selected[0].hitState"
-                :type="tagType"
-                disable-transitions
-                @close="deleteTag($event, selected[0])"
-              >
-                <span :class="nsSelect.e('tags-text')" :style="tagTextStyle">
-                  {{ selected[0].currentLabel }}
-                </span>
-              </el-tag>
-              <el-tag
-                v-if="selected.length > 1"
-                :closable="false"
-                :size="collapseTagSize"
-                :type="tagType"
-                disable-transitions
+              <span
+                :class="[
+                  nsSelect.b('tags-wrapper'),
+                  { 'has-prefix': prefixWidth && selected.length },
+                ]"
               >
-                <el-tooltip
-                  v-if="collapseTagsTooltip"
-                  :disabled="dropMenuVisible"
-                  :fallback-placements="['bottom', 'top', 'right', 'left']"
-                  :effect="effect"
-                  placement="bottom"
-                  :teleported="teleported"
+                <el-tag
+                  v-for="item in selected.slice(0, maxCollapseTags)"
```

Comment:
@kooriookami can we extract this proportion into a computed and name it `tagsToShow` or something?

Reply:
I will add to select-v2 after this PR approved